cmd/skr/main.go

```diff
@@ -202,6 +202,7 @@ func setupServer(certState *attest.CertState, identity *common.Identity, uvmInfo
 	server.Use(httpginendpoints.RegisterGlobalStates(certState, identity, uvmInfo))
 	server.GET("/status", httpginendpoints.GetStatus)
 	server.POST("/attest/raw", httpginendpoints.PostRawAttest)
+	server.POST("/attest/combined", httpginendpoints.PostCombinedAttest) // fetches uvm reference info, certs and attestation report in a form suitable for the Ad Selection API KMS

```

Comment:
Can you add a test script for this endpoint in /docker/skr/tests and then update the example ARM template to have another container to call the script?